tests/raftstore/cluster.rs

```diff
@@ -313,24 +317,32 @@ impl<T: Simulator> Cluster<T> {
                    reqs: Vec<Request>,
                    timeout: Duration)
                    -> RaftCmdResponse {
-        let mut try_cnt = 1;
-        loop {
+        for _ in 0..10 {
             let mut region = self.get_region(key);
             let region_id = region.get_id();
             let req = new_request(region_id, region.take_region_epoch().clone(), reqs.clone());
-            let resp = self.call_command_on_leader(region_id, req, timeout).unwrap();
-            if resp.get_header().has_error() {
-                if self.refresh_leader_if_needed(&resp, region_id) {
-                    warn!("seems leader changed, let's retry");
-                    continue;
-                } else if try_cnt == 1 && resp.get_header().get_error().has_stale_epoch() {
-                    warn!("seems split, let's retry");
-                    try_cnt += 1;
+            let result = self.call_command_on_leader(region_id, req, timeout);
+            match result {
```

Comment:
it's a bit complex here, I want to handle Ok and some of Err case.

is that looks like:

        if let Ok(xxx) = result {
        }
        if let Err(e) = result {
                if let Error::Timeout(xxx) = e {
                }
        }

will compiler complain result moved?